libcob/common.h

```diff
@@ -1272,7 +1273,7 @@ typedef struct __cob_file_key {
 
 
 /* File version (likely can be removed from cob_file in the future) */
-#define	COB_FILE_VERSION	1
+#define	COB_FILE_VERSION	2
```

Comment:
no need for an updated file version here - only later bits are added and the default is unchanged